editor/code_editor.cpp

```diff
@@ -332,6 +332,18 @@ void FindReplaceBar::_update_results_count() {
 	}
 }
 
+void FindReplaceBar::_update_matches_label() {
+
+	if (search_text->get_text().empty() || results_count == -1) {
+		matches_label->hide();
+	} else {
+		matches_label->show();
+
+		matches_label->add_color_override("font_color", results_count > 0 ? Color(1, 1, 1) : EditorNode::get_singleton()->get_gui_base()->get_color("error_color", "Editor"));
+		matches_label->set_text(vformat(TTR(results_count == 1 ? "%d match." : "%d matches."), results_count));
```

Comment:
That's not going to work, `TTR()` is a macro and extracts its contents for localization, but doesn't parse the code.

You should use `vformat(results_count == 1 ? TTR("%d match.") : TTR("%d matches."), results_count)`.